### DISCUSSION TITLE
Foreword - CSDIY.wiki

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#253] [Announcements]
# Foreword - CSDIY.wiki

CS自学指南

https://csdiy.wiki/en/

**Comment by Baaotraan0407** [comment]
請問, 可以在那裡找和自己學assembly資料?
I wanna look for some assembly materials, can you please help me suggest some sources ?

**Comment by nuhuzh** [comment]
Thank you so much for this great resource. Shi-shi… 
Was wondering if you perhaps had any recommendation for what order these should be completed in by those of us doing this self-taught

**Reply by teiyou416** [reply]
I think cs61a/b and 'missing semester of your cs education'by mit is great and everyone should try it once.Also cs144 and cs188 is recommanded.

**Reply by nuhuzh** [reply]
Thank you so much for the response. Very much appreciated.

**Comment by Arickkaa** [comment]
看这些课需要视频吗，还是看一下文章就直接做lab呢

**Reply by teiyou416** [reply]
我除了cs188大多都没看视频，直接做lab应该也可以。不过这可能需要你有比较好的代码基础，不然可能写lab的时候会没有头绪？

**Comment by axlzzd226** [comment]
问一下，Berkeley的课程网站现在是进不去了吗？是要Berkeley的校园网吗？
Have a question, it seems to need logging in before entering the website of Berkeley? Does anybody  have others methods to visit the course web of Berkeley?

**Reply by teiyou416** [reply]
我这里好像还是可以进去的，2018年的网站：https://berkeleyai.github.io/cs188-website/ 2023年的网站：https://inst.eecs.berkeley.edu/~cs188/fa23/

**Reply by axlzzd226** [reply]
> 我这里好像还是可以进去的，2018年的网站：https://berkeleyai.github.io/cs188-website/ 2023年的网站：https://inst.eecs.berkeley.edu/~cs188/fa23/

不过还是十分感谢的❤

**Reply by axlzzd226** [reply]
> 我这里好像还是可以进去的，2018年的网站：https://berkeleyai.github.io/cs188-website/ 2023年的网站：https://inst.eecs.berkeley.edu/~cs188/fa23/

确实，但是有些Berkeley的网站例如UCB EE16A&B: Designing Information Devices and Systems I&II [EE16A](https://inst.eecs.berkeley.edu/~ee16a/su20/)就需要登录其CAS才行。

**Comment by wolfii-ona** [comment]
D+1 of actually learning CS

**Comment by Malam9** [comment]
There is a new version of 61A each term. Which version of the course did you take? A few years ago, Berkeley took down their YouTube videos due to them not complying with accessibility laws (basically it would have been too expensive for them to go back and make captions for all the videos), so now lecture videos aren't available. How does that affect its suitability as a first CS course for people with no prior CS experience?
